spec/features/visitor_can_create_account_spec.rb

```diff
@@ -26,6 +26,10 @@
 
       fill_in "Username", with: 'rossedfort'
       fill_in "Password", with: 'rickross'
+      fill_in 'First name', with: 'Ross'
+      fill_in 'Last name', with: 'Edfort'
+      fill_in 'Email address', with: 'email_address@fake.com'
+      fill_in 'Phone number', with: '101-101-1001'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.